app/actions/player.js

```diff
@@ -6,6 +6,9 @@ export const PAUSE_PLAYBACK = 'PAUSE_PLAYBACK';
 export const UPDATE_PLAYBACK_PROGRESS = 'UPDATE_PLAYBACK_PROGRESS';
 export const UPDATE_SEEK = 'UPDATE_SEEK';
 export const UPDATE_VOLUME = 'UPDATE_VOLUME';
+export const TOGGLE_MUTE = 'TOGGLE_MUTE';
+export const MUTE_ON = 'MUTE_ON';
+export const MUTE_OFF = 'MUTE_OFF';
```

Comment:
Wouldn't "MUTE" and "UNMUTE" be the better name for these constants? Just a suggestion...

Reply:
Agree, thanks for the feedback. I'll update the values.

Reply:
Yeah, in general we want action types to be verbs ("do thing" as opposed to "thing on")